### PR TITLE
Added output options

### PR DESCRIPTION
**Pull request by TomSmartBishop**
Hi, I just added two options to change the output extension via the settings a bit

### CODE REVIEW

File: jinja2_standalone_compiler/__init__.py [hunk 1]
```diff
@@ -46,12 +49,14 @@ def main(path, settings=None):
             continue
 
         html_template, _ = os.path.splitext(jinja_template)
-        html_template = '{}.html'.format(html_template)
+        if output_options['remove_double_extension']:
```
Comment (filwaitman) [inline jinja2_standalone_compiler/__init__.py]: I didn't try this, but I guess this is unsafe and may raise a `KeyError`. What about `output_options.get('remove_double_extension')`?

File: jinja2_standalone_compiler/__init__.py [hunk 2]
```diff
@@ -46,12 +49,14 @@ def main(path, settings=None):
...
+            html_template, _ = os.path.splitext(html_template)
+        html_template = '{}{}'.format(html_template, output_options['extension'])
```
Comment (filwaitman) [inline jinja2_standalone_compiler/__init__.py]: Here extension should default to `.html`, so I guess `output_options.get('extension', '.html)` is better. What do you think?

**Comment by filwaitman** [comment]
Thanks for the PR, @TomSmartBishop! I appreciated the idea, but I have a few concerns about the code. Could you take a look?

**Comment by TomSmartBishop** [comment]
Hey,  I'm perfectly fine changing to your suggestions, since I have no python experience. I just changed the code so that it works for my project and thought it might be a nice extension (and I could use your package without customization on my side).

File: jinja2_standalone_compiler/__init__.py (hunk 1 above)
Comment (TomSmartBishop) [inline jinja2_standalone_compiler/__init__.py]: What does .get(..) return if the key is not present?

Comment (TomSmartBishop) [inline jinja2_standalone_compiler/__init__.py]: I guess output_options.get('remove_double_extension', False) might be better, right?

File: jinja2_standalone_compiler/__init__.py (hunk 2 above)
Comment (TomSmartBishop) [inline jinja2_standalone_compiler/__init__.py]: Sure, why not...

**Comment by filwaitman** [comment]
Now it sounds great!

Thanks, dude!

**Pull request merged**